### ISSUE TITLE
 The app doesn´t play any songs after certain point

### ISSUE THREAD

**Issue by zEzequR** [Venipa/ytmdesktop2#153]
### Operating system

 Microsoft Windows [Versión 10.0.26100.268]

### YouTube Music Desktop App Version

v0.14.5-rc1

### Bug description

Everytime when I try to reproduce a song, it loads until certain point and then it freezes and keeps loading like I don´t have internet until youtube throws me this error message
"An error has occurred. Please refresh the page or try again later."
Hope the developers can help me with this issue.
Thanks!

**Comment by notnightwolf** [comment]
This happens to me since yesterday too...

**Comment by 42301** [comment]
Brought this account back from the grave to say it happens to me too, it's always between 0:59 and 1:05 while a song plays.
Funnily enough, I've only begun noticing it after I stopped paying for premium, maybe it has something to do with ads? Just an idea, I know nothing about programs and such.

**Comment by 42301** [comment]
> Brought this account back from the grave to say it happens to me too, it's always between 0:59 and 1:05 while a song plays. Funnily enough, I've only begun noticing it after I stopped paying for premium, maybe it has something to do with ads? Just an idea, I know nothing about programs and such.

Adding onto this because after I renewed my premium subscription, the issue went away, it has to have something to do with youtube music premium and/or ads.

**Comment by Venipa** [comment]
Yes i was wondering to how this could happen, im on yt premium and could not reproduce it.
Probably has to do something with the ad skipper.
I'll look into it soon

**Comment by NobinGJ** [comment]
I also have the same problem and if as they say the problem is the ads, when YT Music wants to load an ad it runs into the blocker that the program brings and it seems that it fails or fails to block well and throws the error, so until they improve the ad blocker we will remain like this

**Comment by Venipa** [comment]
recent update should fix this issue @zEzequR @notnightwolf could you report back if this is still happening?
thanks 🤗

**Comment by eddcm123** [comment]
> recent update should fix this issue [@zEzequR](https://github.com/zEzequR) [@notnightwolf](https://github.com/notnightwolf) could you report back if this is still happening? thanks 🤗

Hey I'm updated and I'm still experiencing the same here

https://github.com/user-attachments/assets/d84031c8-64a1-4825-99d6-da6a5abc2847

After around 1 minute it stops and does never pass that 
 Help!

**Comment by Venipa** [comment]
duplicate - check #155
